tests/unit/amo/components/TestCategory.js

```diff
@@ -436,6 +438,92 @@ describe(__filename, () => {
     });
   });
 
+  it('renders the themes shelves with the ADDON_TYPE_THEMES_FILTER filter if static theme is enabled', () => {
+    _categoriesFetch();
+    _categoriesLoad();
+    _getLanding();
+    _loadLanding();
+
+    const fakeConfig = getFakeConfig({ enableStaticThemes: true });
+
+    const root = render(
+      { _config: fakeConfig },
+      { autoDispatchCategories: false },
+    );
+
+    const landingShelves = root.find(LandingAddonsCard);
+
+    expect(landingShelves.at(0)).toHaveProp('footerLink', {
+      pathname: `/search/`,
+      query: {
+        addonType: ADDON_TYPE_THEMES_FILTER,
+        category: fakeCategory.slug,
+        featured: true,
+      },
+    });
+
+    expect(landingShelves.at(1)).toHaveProp('footerLink', {
+      pathname: '/search/',
+      query: {
+        addonType: ADDON_TYPE_THEMES_FILTER,
+        category: fakeCategory.slug,
+        sort: SEARCH_SORT_TOP_RATED,
+      },
+    });
+
+    expect(landingShelves.at(2)).toHaveProp('footerLink', {
+      pathname: '/search/',
+      query: {
+        addonType: ADDON_TYPE_THEMES_FILTER,
+        category: fakeCategory.slug,
+        sort: SEARCH_SORT_TRENDING,
+      },
+    });
+  });
+
+  it('renders the themes shelves with the ADDON_TYPE_THEME filter if static theme is disabled', () => {
+    _categoriesFetch();
+    _categoriesLoad();
+    _getLanding();
+    _loadLanding();
+
+    const fakeConfig = getFakeConfig({ enableStaticThemes: false });
```

Comment:
This works only because it is `false` by default, but not because you have a specific config.

Reply:
right, i just thought it was better to be explicit since this is the what the test describes, no?

Reply:
What I mean is that you could set `enableStaticThemes` to `true`, the test would not fail because as far as I know, `config` is not a prop of `Category`.

Reply:
yeh i get it now 🤦‍♂️  thanks:)